packages/docs-site/src/library/pages/components/forms/input.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Input
+title: Text Input
```

Comment:
Nit picking - should this be another PR? I guess it's a non-consequential change.

Reply:
I changed it in this PR to differentiate it from the Number input. I though it was part of this pr. Good to see though that you are catching this :)